src/install.rs

```diff
@@ -40,6 +40,14 @@ pub fn install(config: &InstallConfig) -> Result<()> {
         }
     }
 
+    // if an Ignition config is provided, check upfront that we can open it
+    if let Some(ignition) = &config.ignition {
+        OpenOptions::new()
+            .read(true)
+            .open(ignition)
+            .chain_err(|| format!("opening source Ignition config {}", ignition))?;
```

Comment:
What about keeping this around as an `Option<File>` and passing it directly to `write_disk`?

(Not strictly relevant in this case, but current approach is a pattern prone to TOCTOU, thus I try to avoid it)

Reply:
Thanks, done!